cinema/views.py

```diff
@@ -57,7 +58,7 @@
 ):
     queryset = CinemaHall.objects.all()
     serializer_class = CinemaHallSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` is commented out in `CinemaHallViewSet`. If authentication is required for these endpoints, consider uncommenting this line.